content/en/agent/guide/network.md

```diff
@@ -343,6 +343,16 @@ See [default NTP targets][2].
 {{% /tab %}}
 {{< /tabs >}}
 
+## Configure ports
+
+If you need to change an inbound port because the default port is already in use by an existing service on your network, use the `datadog.yaml` configuration file. You can find most of the ports in the **Advanced Configuration** section of the file:
```

Comment:
```suggestion
If you need to change an inbound port because the default port is already in use by an existing service on your network, edit the `datadog.yaml` configuration file. You can find most of the ports in the **Advanced Configuration** section of the file:
```